website/_config.yml

```diff
@@ -0,0 +1,83 @@
+# Hexo Configuration
+## Docs: https://hexo.io/docs/configuration.html
+## Source: https://github.com/hexojs/hexo/
+
+# Site
+title: Hexo
```

Comment:
Even if this isn't used anywhere, we should probably update it with better information (or at least remove the placeholder data).